src/Compilers/CSharp/Portable/FlowAnalysis/EmptyStructTypeCache.cs

```diff
@@ -213,6 +213,13 @@ private FieldSymbol GetActualField(Symbol member, NamedTypeSymbol type)
                 case SymbolKind.Event:
                     var eventSymbol = (EventSymbol)member;
                     return (!eventSymbol.HasAssociatedField || ShouldIgnoreStructField(eventSymbol, eventSymbol.Type)) ? null : eventSymbol.AssociatedField.AsMember(type);
+                case SymbolKind.Property:
+                    // Backing field for semi auto props are not included in GetMembers.
+                    if (member is SourcePropertySymbol { BackingField: SynthesizedBackingFieldSymbol { IsCreatedForFieldKeyword: true } backingField})
```

Comment:
>member is SourcePropertySymbol { BackingField: SynthesizedBackingFieldSymbol { IsCreatedForFieldKeyword: true } backingField} [](http://example.com/codeflow?start=24&length=125)

Is the field going to be created here if it hasn't been yet, but is supposed to exist?